rasa/core/policies/policy.py

```diff
@@ -67,6 +68,8 @@ def featurize_for_training(
         self,
         training_trackers: List[DialogueStateTracker],
         domain: Domain,
+        interpreter: Optional[RasaCoreInterpreter],
+        output_path,
```

Comment:
please add a comment that `output_path` should be removed. It's temporary because core interpreter is persisted in a train for now

Reply:
Added the comment below